lightkurve/correctors.py

```diff
@@ -672,15 +680,18 @@ def correct(self, aperture_mask=None, cadence_mask=None, gp_timescale=30,
         self.flux_err = self.flux_err[nanmask]
         self.time = self.time[nanmask]
 
+        # parse PLD aperture mask
+        pld_pixels = self.tpf._parse_aperture_mask(pld_aperture)
```

Comment:
:point_up: It may be best to consistently use the suffix `_mask` in all variables that contain a ... mask.

Reply:
(A reader of the code may be tempted to think that `pld_pixels` contain pixel values otherwise.)